src/Product.php

```diff
@@ -207,6 +207,10 @@ public static function woocommerce_available_variation(array $variation_data, \W
    * Displays the shop stock-status component on the front-end.
    */
   public static function woocommerce_product_meta_start() {
+    if (get_the_terms(wc_get_product()->get_id(), 'product_cat')[0]->name === 'Ausstellungsstücke') {
```

Comment:
1. The `global $product` is already used below. You can also use it.
2. WordPress has a dedicated API function to test whether a post has a term; see https://rudrastyh.com/woocommerce/if-product-in-category.html
3. This is a shared plugin. The category name is a magic value. We do not need to make it configurable for now, but can you move the value into a class constant `Product::CATEGORY_EXCLUDED`?